### DISCUSSION TITLE
Wording: Plugins or Scripts

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [WinMerge/winmerge#1315] [Q&A]
Is there a difference, or is it (should it be) the same?
![grafik](https://user-images.githubusercontent.com/13277944/165944541-1bb65e4a-9d9e-49fa-a888-8593085e6d10.png)

**Comment by sdottaka** [comment]
Perhaps I don't understand your question exactly, but "Scripts" means "Editor script plugins".
There are three types of plug-ins as follows.

-Unpacker plugins
- Prediffer plugins
- Editor script plugins (referred to as Editor complement in the manual)

For this reason, Scripts cannot simply be referred to as Plugins.

**Comment by Pierre-de-la-Verre** [comment]
I'm a little bit confused too, but if "Scripts" are a sub-group of plugins, then "Suggested plugins" can not be a sub-menu of Scripts. Maybe "Suggested scripts" is better?
But it is not really important and I'm not sure about it ..
Thanks.
